src/components/content/order/migrate/MigratingServices.tsx

```diff
@@ -0,0 +1,375 @@
+                setTip(OrderSubmitFailed(error));
+                setDestroying(false);
+                setRequestSubmitted(false);
+            });
+    };
+
+    function waitingServiceDestroy(uuid: string, timeout: number, date: Date) {
+        setTip(
+            OrderSubmitResult(
+                'Destroying, Please wait... [' +
+                    Math.ceil((new Date().getTime() - date.getTime()) / 1000).toString() +
+                    's]',
+                uuid,
+                'success'
+            )
+        );
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        ServiceService.getDeployedServiceDetailsById(uuid, localStorage.getItem(usernameKey)!)
+            .then((response) => {
+                setRequestSubmitted(false);
+                if (response.serviceState === ServiceDetailVo.serviceState.DESTROY_SUCCESS) {
+                    setTip(
+                        OrderSubmitResult(
+                            ProcessingStatus(response, OperationType.destroy as OperationType),
+                            uuid,
+                            'success'
+                        )
+                    );
+                    setDestroying(false);
+                    TipClear();
+                    getCloseStatus(true);
+                } else if (response.serviceState === ServiceDetailVo.serviceState.DESTROY_FAILED) {
+                    setTip(
+                        OrderSubmitResult(
+                            ProcessingStatus(response, OperationType.destroy as OperationType),
+                            uuid,
+                            'error'
+                        )
+                    );
+                    setDestroying(false);
+                    TipClear();
+                } else {
+                    setTimeout(() => {
+                        waitingServiceDestroy(uuid, timeout - waitServicePeriod, date);
+                    }, waitServicePeriod);
+                }
+            })
+            .catch((e: Error) => {
+                setTip(OrderSubmitResult('Destroy failed:' + e.message, uuid, 'error'));
+                setDestroying(false);
+                TipClear();
+                setRequestSubmitted(false);
+            });
+    }
+
+    const steps = [
+        {
+            title: 'Export data',
+            content: <ExportServiceData />,
+            description: 'Export service data.',
+        },
+        {
+            title: 'Select a destination',
+            content: (
+                <SelectDestination
+                    userAvailableServiceVoList={userAvailableServiceVoList}
+                    getSelectedParameters={getSelectedParameters}
+                    currentCsp={selectCsp}
+                    currentArea={selectArea}
+                    currentRegion={selectRegion}
+                    currentFlavor={selectFlavor}
+                />
+            ),
+            description: 'Select a destination for deployment.',
```

Comment:
Select a destination for migrating the existing deployment.

Reply:
fixed